experimental/examples/bubblebench/editable-shared-tree/src/AppState.ts

```diff
@@ -0,0 +1,100 @@
+/*!
```

Comment:
we generally go with lowercase file names for our typescript files

Reply:
I recommend addressing the rest of the commends before doing any renames as it will make it much easier to access the comments.